### PR TITLE
Add aspects to curriculum/javascript/core/recipes-i

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/recipes-i/using-the-double-tilde-game.md?ref=5adcaf61562d25d1f5b07d0c18b017dd1c477275

File: javascript/core/recipes-i/format-currency-in-javascript.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ links:
 
 
 aspects:
-  - introduction
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/recipes-i/format-currency-in-javascript.md]: I would revert this change, it makes sense as `introduction`.

File: javascript/core/recipes-i/js-practice-math-round.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ standards:
   javascript.standard-library.1: 1000
   javascript.write-expressions.0: 1000
 
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/recipes-i/js-practice-math-round.md]: This is `introduction`, most languages have this function, so I don't think it makes sense as a `workout`.

File: javascript/core/recipes-i/js-practice-number-tofixed.md [hunk 3]
```diff
@@ -11,6 +11,10 @@ type: exercise
 
 link: http://www.codewars.com/kata/57256064856584bc47000611
 linkType: codewars
+
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/recipes-i/js-practice-number-tofixed.md]: This is the same material as the currency formatter, so it should also be `introduction`.

File: javascript/core/recipes-i/js-practice-number-tostring.md [hunk 4]
```diff
@@ -11,6 +11,10 @@ type: exercise
 
 link: http://www.codewars.com/kata/57238ceaef9008adc7000603
 linkType: codewars
+
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/recipes-i/js-practice-number-tostring.md]: Revert to `introduction`

File: javascript/core/recipes-i/quick-way-to-format-integers-with-a-fixed-number-of-digits.md [hunk 5]
```diff
@@ -34,8 +34,9 @@ links:
 
 
 aspects:
-  - introduction
   - workout
+  - obscura
+  - deep
```
Comment (ryanwarsaw) [inline javascript/core/recipes-i/quick-way-to-format-integers-with-a-fixed-number-of-digits.md]: I would leave `introduction`, and remove `deep`. It's fine with `obscura` though.

File: javascript/core/recipes-i/round-numbers-to-n-decimals.md [hunk 6]
```diff
@@ -17,6 +17,9 @@ standards:
 
   javascript.evaluate-expressions.3: 10
 
+aspects:
```
Comment (ryanwarsaw) [inline javascript/core/recipes-i/round-numbers-to-n-decimals.md]: Should be `introduction` so it's consistent with currency formatter.

File: javascript/core/recipes-i/using-the-double-tilde-game.md [hunk 7]
```diff
@@ -13,6 +13,9 @@ type: fillTheGap
 
 category: tip
 
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/recipes-i/using-the-double-tilde-game.md]: Fair, I would also add `introduction`.

File: javascript/core/recipes-i/using-the-double-tilde.md [hunk 8]
```diff
@@ -25,6 +25,9 @@ tags:
 
   - double-tilde
 
+aspects:
+  - workout
+  - deep
```
Comment (ryanwarsaw) [inline javascript/core/recipes-i/using-the-double-tilde.md]: Remove `deep` aspect, and add `introduction`
